gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/IosResources.kt

```diff
@@ -113,7 +113,7 @@ private fun Framework.isCocoapodsFramework() = name.startsWith("pod")
 private fun Framework.getFinalResourcesDir(): Provider<Directory> {
     val providers = project.providers
     return if (isCocoapodsFramework()) {
-        project.layout.buildDirectory.dir("compose/ios/$baseName/$IOS_COMPOSE_RESOURCES_ROOT_DIR/")
+        project.layout.buildDirectory.dir("compose/cocoapods/$IOS_COMPOSE_RESOURCES_ROOT_DIR/")
```

Comment:
Does it work for case with multiple pods/frameworks now?

Reply:
we cannot produce more then one a framework from a gradle module with a podspec